docs/components.md

```diff
@@ -122,6 +122,7 @@ The distribution offers support for the following components.
 
 | Extensions                                                                                                                          | Stability |
 | :---------------------------------------------------------------------------------------------------------------------------------- | :-------- |
+| [headerssetterextension](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/headerssetterextension)                 | [alpha]   |
```

Comment:
This should be added in alphabetical order to make searching through the table easier.